### ISSUE TITLE
Aggregated reports for gradle composite build not generated

### ISSUE THREAD

**Issue by ShrutiTalewad** [Kotlin/kotlinx-kover#123]
Currently I am able to generate aggregated reports for individual builds but I am trying to get aggregated reports which includes reports of all builds 

Using latest version 0.5.0-RC2 
Tried with includes on koverMergedHtmlReport like below in build.gradle file of parent build (build-1) but doesn't seem to work

    koverMergedHtmlReport {
        dependsOn(gradle.includedBuild("build-2").task(":koverMergedHtmlReport"))
        includes = listOf("com.example1.*", "com.foo.*")   // This is the fully qualified class name of packages in second build}

**Comment by shanshin** [comment]
Hi,
composing builds are not supported by Kover and there are no short-term plans to add such a feature.

**Comment by slm** [comment]
I get this error when I try to apply Kover with composing build.

```
Failed to query the value of task ‘:common-ui:koverHtmlReport’ property ‘files$kover’.
> Type org.jetbrains.kotlin.gradle.dsl.KotlinAndroidProjectExtension not present
```

Is it the reason? If it is when do you planning the support it?

**Comment by shanshin** [comment]
Closed in prior of #487
